content/inheritance/reading/cats-inheritance/_index.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "A Tale of Two Cats"
+date: 2023-01-31T15:39:38-06:00
+draft: false
+weight: 3
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+<!-- TODO: Link to repository -->
+
+Let’s revisit our `Cat` and `HouseCat` friends. Check out this [repository](LINK) for the code for the lesson. In Visual Studio, open the solution and examine the two classes inside the Cats project.
```

Comment:
Code link?

Reply:
rephrased and redirected students to the examples repo. thanks!